common/mayastor/nexus_ha/nexus_ha.go

```diff
@@ -917,3 +918,28 @@ func GetOnlineReplicasCount(uuid string) (int, error) {
 func VerifyMayastorAndPoolReady() (bool, error) {
 	return k8stest.VerifyMayastorAndPoolReady(DefTimeoutSecs)
 }
+
+// Prevent mayastor pod from running on the given node.
+func SuppressMayastorPodOnNode(nodeName string, timeout int) error {
```

Comment:
These 2 functions should be in the k8stest package

Reply:
moved it to k8stest package https://github.com/openebs/openebs-e2e/compare/040b7397c401d1e1bc9a901cb33a02ec6a9152f2..9b31d5f5bf82cfad5a855b224d3c14449871eea4